README.md

```diff
@@ -94,10 +94,21 @@ As mentioned above, you'll need to register your client ID. We recommend you tha
 To define your app credentials, you'll need to create an instance of `Spotify::Accounts`:
 
 ```ruby
-@accounts = Spotify::Accounts.new
-@accounts.client_id = ENV["SPOTIFY_CLIENT_ID"]
-@accounts.client_secret = ENV["SPOTIFY_CLIENT_SECRET"]
-@accounts.redirect_uri = ENV["SPOTIFY_REDIRECT_URI"]
+@accounts = Spotify::Accounts.new({
+  client_id: "spotify client ID",
+  client_secret: "spotify client secret",
+  redirect_uri: "redirect URI"
+})
```

Comment:
There's a typo here (missing a `}` on line 101).

Also, I think we can format this like:

```ruby
accounts = Spotify::Accounts.new(client_id: "spotify client id",
          												 client_secret: "spotify client secret",
													     redirect_uri: "spotify redirect uri")
```